src/format/ipynb/format-ipynb.ts

```diff
@@ -46,6 +46,7 @@ export function ipynbFormat(): Format {
           join("templates", "title-block.md"),
         );
 
+        // FIXME: how do we get the cwd from here?
```

Comment:
I _think_ that `_input` above (passed to `formatExtras`) is a full path to the input. I think `cwd` might be better named as `inputDir` rather than `cwd` since these paths should be resolved against input relative paths (rather than cwd relative paths).